### ISSUE TITLE
Word Bloat Suggestion

### ISSUE THREAD

**Issue by Akerszp** [belthesar/MalmstoneXPCalculator#1]
The results are too wordy with a lot of redundant words like 'matches' and type of PVP. 
I have a suggestion to make it less wordy and more easy to read.

```
Crystaline Conflict
	x Wins
	x Loss
	
Frontline
	x 1st Place
	x 2nd Place
	x 3rd Place
	
	x Daily 1st
	x Daily 2nd
	x Daily 3rd
	
Rival Wings
	x Wins
	x Loss
```

**Comment by belthesar** [comment]
Hey, thanks for the suggestion! Good feedback. I'll look at seeing what I can do to condense. I do want to make sure it looks reasonably clear that this isn't the list of things to do, but a list of things one can do, which was why I took the previous approach.

**Comment by belthesar** [comment]
It took downtime before Dawntrail to come back to this, but I've taken your feedback into account with some language and formatting changes. Thanks for your input!
